modules/ROOT/pages/ogm/api-reference/index.adoc

```diff
@@ -1,5 +1,6 @@
 [[ogm-api-reference]]
 = API Reference
+:page-aliases: ogm/reference.adoc
```

Comment:
This page is already linked, it has the three versions options in the dropdown menu

Reply:
true, but it's the "aggregate" page at the bottom of the navigation in 3.x, instead of the first (which is this API reference).

Checked locally, there's no explicit page-alias for "aggregate" either, so adding the page-alias here won't change anything (again, why is that?)

Reply:
It might be the same as the previous comment, that in 5.x this page is already an alias to another one... so that's why I think we should only work in the most recent branch and the older ones will catch up? Or they should, like we were discussing... otherwise it might create a conflict between branches